packages/app-store/routing-forms/playwright/tests/basic.e2e.ts

```diff
@@ -259,11 +261,17 @@ test.describe("Routing Forms", () => {
         responses.push(rowResponses);
       }
 
-      expect(headers).toEqual(["Test field", "Multi Select(with Legacy `selectText`)", "Multi Select"]);
+      expect(headers).toEqual([
+        "Test field",
+        "Multi Select(with Legacy `selectText`)",
+        "Multi Select",
+        "Legacy Select",
+        "Select",
+      ]);
       expect(responses).toEqual([
-        ["event-routing", "", ""],
-        ["external-redirect", "", ""],
-        ["custom-page", "", ""],
+        ["event-routing", "Option-2", "Option-2", "Option-2", "Option-2"],
+        ["external-redirect", "Option-2", "Option-2", "Option-2", "Option-2"],
+        ["custom-page", "Option-2", "Option-2", "Option-2", "Option-2"],
       ]);
```

Comment:
Verifying data submission for both multiseelct and select in old and new options format.